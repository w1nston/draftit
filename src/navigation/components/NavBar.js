import React, { Fragment } from 'react';
import { Link } from 'react-router-dom';
import { css } from 'emotion';

const navStyle = css`
  display: flex;
  justify-content: space-around;
`;

const NavBar = () => (
  <Fragment>
    <nav className={navStyle}>
      <Link to="/">Posts</Link>
      <Link to="/new">New post</Link>
      <Link to="/logout">Logout</Link>
    </nav>
    <hr />
  </Fragment>
);

export default NavBar;
