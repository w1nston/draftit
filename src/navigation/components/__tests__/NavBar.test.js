import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import NavBar from '../NavBar';

describe('NavBar', () => {
  it('renders', () => {
    const component = shallow(<NavBar />);
    expect(toJson(component)).toMatchSnapshot();
  });
});
