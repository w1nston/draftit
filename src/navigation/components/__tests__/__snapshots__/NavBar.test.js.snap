// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar renders 1`] = `
<React.Fragment>
  <nav
    className="css-1sulisw"
  >
    <Link
      replace={false}
      to="/"
    >
      Posts
    </Link>
    <Link
      replace={false}
      to="/new"
    >
      New post
    </Link>
    <Link
      replace={false}
      to="/logout"
    >
      Logout
    </Link>
  </nav>
  <hr />
</React.Fragment>
`;
